model/task_lifecycle.go

```diff
@@ -153,7 +153,7 @@ func SetActiveStateById(id, user string, active bool) error {
 // originalStepbackTask is only specified if we're first activating the generator for a generated task.
 func activatePreviousTask(taskId, caller string, originalStepbackTask *task.Task) error {
 	// find the task first
-	t, err := task.FindOne(task.ById(taskId))
+	t, err := task.FindOneId(taskId)
```

Comment:
this doesn't seem like it would need test results

Reply:
This is the bad naming I was talking about—`task.FindOneId` does not merge test results.

Reply:
Oh haha nice. Can we add NoMerge then to be consistent with the other no merge functions?

Reply:
It is misleading, I know. I will try and fix it in the next PR.

Reply:
Or else `FindOneNoMerge(task.ById` is probably preferable.

Reply:
Yeah, since I think I am going to get rid of merge functions in the next PR I think I would rather leave it as is.